.rules/default-exports.md

```
@@ -0,0 +1,36 @@
+Unless explicitly required by the framework, do not use default exports.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)